netci.groovy

```diff
@@ -59,6 +59,47 @@ def addPushJob(String project, String branch, String os, String configuration)
 };
 
 [true, false].each { isPR ->
+  ["RHEL7.2"].each { os->
+    ["Release", "Debug"].each { configuration ->
+
+      def shortJobName = "${os}_Offline_${configuration}";
+      def contextString = "${os} ${configuration}";
```

Comment:
Is this meant to be the same as the non-offline build trigger phrases?  As is I think "test RHEL7.2 Release" kicks both the offline and online builds.

Reply:
No.  It should include Offline.  I'll change it to "${os} Offline ${configuration}"